codec/common/src/cpu.cpp

```diff
@@ -359,6 +359,32 @@ uint32_t WelsCPUFeatureDetect (int32_t* pNumberOfLogicProcessors) {
     return get_cpu_flags_from_cpuinfo();
 }
 
+#elif defined(__loongarch__) && defined(__linux__)
+/* for loongson */
+#define LOONGARCH_CFG2 0x02
+#define LOONGARCH_CFG2_LSX  (1<<6)
+#define LOONGARCH_CFG2_LASX (1<<7)
+
+static uint32_t get_cpu_flags_from_cpuinfo(void) {
```

Comment:
Rename this function, it's using cpucfg here.

Reply:
Thanks, Will update.